src/databricks/labs/remorph/helpers/reconcile_config_utils.py

```diff
@@ -0,0 +1,390 @@
+import json
+import logging
+import webbrowser
+
+from databricks.connect import DatabricksSession
+from databricks.labs.blueprint.installation import Installation, SerdeError
+from databricks.labs.blueprint.tui import Prompts
+from databricks.labs.remorph.config import TableRecon, ReconcileConfig, get_dialect
+from databricks.labs.remorph.reconcile.connectors.source_adapter import create_adapter
+from databricks.labs.remorph.reconcile.constants import SourceType
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors.platform import ResourceDoesNotExist, NotFound, PermissionDenied
+
+logger = logging.getLogger(__name__)
+
+README_RECON_CONFIG_NOTEBOOK = """# Databricks notebook source
+# MAGIC %md
+# MAGIC # Recon Config setup instructions (see [README](readme_link))
+# MAGIC
+# MAGIC Production runs are supposed to be triggered through the following jobs: job_links
+# MAGIC
+# MAGIC **This notebook is overwritten with each `remorph` generate-recon-config.**
+
+# COMMAND ----------
+
+# MAGIC %pip install remorph_VERSION-py3-none-any.whl
+dbutils.library.restartPython()
+
+# COMMAND ----------
+
+from databricks.labs.remorph.config import get_dialect
+from databricks.labs.remorph.reconcile.recon_config import ColumnMapping, Schema, Table
+from databricks.labs.remorph.reconcile.schema_compare import SchemaCompare
+
+src_schema, tgt_schema = schemas["snowflake_databricks_schema"]
+    spark = mock_spark
+    table_conf = Table(
+        source_name="supplier",
+        target_name="supplier",
+        drop_columns=["dummy"],
+        column_mapping=[
+            ColumnMapping(source_name="col_char", target_name="char"),
+            ColumnMapping(source_name="col_array", target_name="array_col"),
+        ],
+    )
+
+schema_compare_output = SchemaCompare(spark).compare(
+    src_schema,
+    tgt_schema,
+    get_dialect("snowflake"),
+    table_conf,
+)
+df = schema_compare_output.compare_df
+"""
+
+
+class ReconcileConfigUtils:
+    def __init__(self, ws: WorkspaceClient, installation: Installation = None, prompts: Prompts = Prompts()):
```

Comment:
```suggestion
    def __init__(self, ws: WorkspaceClient, installation: Installation, prompts: Prompts = Prompts()):
```
don't make things optional when they are required.